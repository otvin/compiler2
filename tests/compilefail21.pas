program compilefail21(output, fizzazle);
{test invalid program parameters}

begin
	writeln('hello world');
end.
