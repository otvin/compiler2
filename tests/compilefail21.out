Invalid program parameter 'fizzazle' in file:tests/compilefail21.pas line:1     column:31 
